index.html

```diff
@@ -14,20 +14,33 @@
 <header>
     <nav class="nav-bar" >
         <div class="logo"><a>John Doe </a> </div>
-      <nav>
+      <nav class="menu" >
            <div class="bar1"></div>
   <div class="bar2"></div>
   <div class="bar3"></div>
       </nav>
+      <ul class="menu-list">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li><img src="./img/Union.png" alt="mail"></li>
+    </ul>
```

Comment:
- [ ] You should add your nav links inside the `a` tag, if you don´t have an available URL you should add the # for the `href` value, like this: `href="#"`
- [ ] [OPTIONAL] To make the transition between nav links and the sections you can use the id section after the #. For instance:
```
<a href=#home-section""></a> // Link to go to home section

<section id="home-section">
</section>
```